src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
Most of the code has been written with the dynamic mode in mind (as I thought it was the best and most used version). For instance, in the static mode, we should not care about counting the number of particles either, but we do.
Distinguishing the modes would imply to:
1) add a new flag on the simulator command line
2) add a switch in the CARRY_ON of Gate.Java to either compute for a duration (dynamic) or a number of particles (static). In the static mode, the job never sends PROGRESS message but directly goes to the "END" section to send a DISCONNECT message. This way, we may not have to modify the code of VIPServer, or only slightly.
     Note: I'm not sure a GATE job has a number of particles as input right now ... Does this mean we have to divide the total number by the number of jobs?